tests/user_commands/tests.py

```diff
@@ -72,6 +74,17 @@ def test_find_command_without_PATH(self):
             if current_path is not None:
                 os.environ['PATH'] = current_path
 
+    def test_discover_commands_in_eggs(self):
+        """
+        Test that management commands can also be loaded from Python eggs.
+        """
+        egg_dir = '%s/eggs' % os.path.dirname(upath(__file__))
+        egg_name = '%s/basic.egg' % egg_dir
+        with extend_sys_path(egg_name):
+            with self.settings(INSTALLED_APPS=['commandegg']):
+                cmds = find_commands(os.path.join(apps.get_app_config('commandegg').path, 'management'))
+        self.assertListEqual(cmds, ['eggcommand'])
```

Comment:
Nitpick: assertEqual calls assertListEqual if both a and b is a list: See https://github.com/python/cpython/blob/master/Lib/unittest/case.py#L389 and https://github.com/python/cpython/blob/master/Lib/unittest/case.py#L777